apps/prairielearn/src/pages/administratorCourseRequests/administratorCourseRequests.js

```diff
@@ -18,8 +25,35 @@ router.get(
   '/',
   asyncHandler(async (req, res) => {
     res.locals.coursesRoot = config.coursesRoot;
-    const result = await sqldb.queryOneRowAsync(sql.get_requests, []);
-    _.assign(res.locals, result.rows[0]);
+    const result = await sqldb.queryRow(
+      sql.get_requests,
+      [],
+      z.object({
+        institutions: z.array(InstitutionSchema),
+        course_requests: z.array(
+          CourseRequestSchema.extend({
+            user_name: z.string(),
+            created_at: DateFromISOString.optional(),
+            approved_by: z.string().optional(),
+            approved_status: z
+              .enum(['pending', 'approved', 'denied', 'creating', 'failed'])
+              .optional(),
+            user_id: z.string().optional(),
```

Comment:
Why is this `optional` and not `nullable`? This is the last one of these I'll comment on; there are probably more. Please check on your own.

Reply:
As noted elsewhere, I was incorrectly using this to modify the `CourseRequestSchema` for values that were not being returned from the query. I was also having issues with incorrect test data that I had entered into the db and was giving me bad testing data. I have now corrected info in my local db and written `CourseRequestRowSchema` that matches the intended results from the query.